README.md

```diff
@@ -203,6 +203,103 @@ for label in (ax.get_xticklabels() + ax.get_yticklabels()):
     
 
 
+### Adding external regressors
+
+```python
+series_xreg = pd.DataFrame(
+    {
+        'ds': pd.date_range(start='1949-01-01', periods=ap_train.size, freq='M'),
+        'y': ap_train
+    },
+    index=pd.Index([0] * ap_train.size, name='unique_id')
+)
+```
+
+```python
+series_xreg['trend'] = np.arange(1, ap_train.size + 1)
+series_xreg['month'] = series_xreg['ds'].dt.month
+series_xreg = pd.get_dummies(series_xreg, columns=['month'], drop_first=True)
+```
+
+```python
+display_df(series_xreg.head())
+```
+
+
+|   unique_id | ds                  |   y |   trend |   month_2 |   month_3 |   month_4 |   month_5 |   month_6 |   month_7 |   month_8 |   month_9 |   month_10 |   month_11 |   month_12 |
+|------------:|:--------------------|----:|--------:|----------:|----------:|----------:|----------:|----------:|----------:|----------:|----------:|-----------:|-----------:|-----------:|
+|           0 | 1949-01-31 00:00:00 | 112 |       1 |         0 |         0 |         0 |         0 |         0 |         0 |         0 |         0 |          0 |          0 |          0 |
+|           0 | 1949-02-28 00:00:00 | 118 |       2 |         1 |         0 |         0 |         0 |         0 |         0 |         0 |         0 |          0 |          0 |          0 |
+|           0 | 1949-03-31 00:00:00 | 132 |       3 |         0 |         1 |         0 |         0 |         0 |         0 |         0 |         0 |          0 |          0 |          0 |
+|           0 | 1949-04-30 00:00:00 | 129 |       4 |         0 |         0 |         1 |         0 |         0 |         0 |         0 |         0 |          0 |          0 |          0 |
+|           0 | 1949-05-31 00:00:00 | 121 |       5 |         0 |         0 |         0 |         1 |         0 |         0 |         0 |         0 |          0 |          0 |          0 |
+
+
+```python
+xreg_test = pd.DataFrame(
+    {
+        'ds': pd.date_range(start='1960-01-01', periods=ap_test.size, freq='M')
+    },
+    index=pd.Index([0] * ap_test.size, name='unique_id')
+)
+```
+
+```python
+xreg_test['trend'] = np.arange(133, ap_test.size + 133)
+xreg_test['month'] = xreg_test['ds'].dt.month
+xreg_test = pd.get_dummies(xreg_test, columns=['month'], drop_first=True)
```

Comment:
I think we should add an intercept or leave the first dummy here.

Reply:
Cool, solved here 5dd8ebb8282a515cce324cf9eeb7a3999b09463c.